.github/workflows/release-v0.yaml

```diff
@@ -6,11 +6,15 @@ on:
       majorRelease:
         required: false
         description: 'Release as version. Example values are: minor, major, 5.6.1, 1.0.0. Leave empty for regular release.'
+      tag:
+        required: false
+        description: 'Tag name. Leave empty for regular release.'
```

Comment:
Nitpicking, but I was confused at first. Let's have description starting with `Prerelease tag name`. Otherwise `1.0.0` is also a tag